```
=================================== FAILURES ===================================
  ________________ TestPodManager.test_fetch_container_since_time ________________
  
  self = <tests.providers.cncf.kubernetes.utils.test_pod_manager.TestPodManager object at 0x7f2aeb2d9f10>
  container_running = <MagicMock name='container_is_running' id='139822292685072'>
  mock_now = <MagicMock name='now' id='139822237497808'>
  
      @mock.patch('pendulum.now')
      @mock.patch('airflow.providers.cncf.kubernetes.utils.pod_manager.container_is_running')
      def test_fetch_container_since_time(self, container_running, mock_now):
          """If given since_time, should be used."""
          mock_pod = MagicMock()
          mock_now.return_value = DateTime(2020, 1, 1, 0, 0, 5, tzinfo=Timezone('UTC'))
          container_running.return_value = False
          self.mock_kube_client.read_namespaced_pod_log.return_value = [b'2021-01-01 hi']
          since_time = DateTime(2020, 1, 1, tzinfo=Timezone('UTC'))
          self.pod_manager.fetch_container_logs(pod=mock_pod, container_name='base', since_time=since_time)
          call_kwargs = self.mock_kube_client.read_namespaced_pod_log.call_args_list[0].kwargs
  >       assert call_kwargs['since_seconds'] == 5
  E       TypeError: tuple indices must be integers or slices, not str
  
  tests/providers/cncf/kubernetes/utils/test_pod_manager.py:309: TypeError
  ----------------------------- Captured stderr call -----------------------------
  INFO  [airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager] hi
  ------------------------------ Captured log call -------------------------------
  INFO     airflow.providers.cncf.kubernetes.utils.pod_manager.PodManager:pod_manager.py:216 hi
```

--- a/airflow/providers/cncf/kubernetes/utils/pod_manager.py
+++ b/airflow/providers/cncf/kubernetes/utils/pod_manager.py
@@ -87,7 +87,7 @@ class PodLoggingStatus:
     """Used for returning the status of the pod and last log time when exiting from `fetch_container_logs`"""
 
     running: bool
-    last_log_time: DateTime
+    last_log_time: Optional[DateTime]
 
 
 class PodManager(LoggingMixin):

--- a/tests/providers/cncf/kubernetes/utils/test_pod_manager.py
+++ b/tests/providers/cncf/kubernetes/utils/test_pod_manager.py
@@ -305,8 +305,8 @@ class TestPodManager:
         self.mock_kube_client.read_namespaced_pod_log.return_value = [b'2021-01-01 hi']
         since_time = DateTime(2020, 1, 1, tzinfo=Timezone('UTC'))
         self.pod_manager.fetch_container_logs(pod=mock_pod, container_name='base', since_time=since_time)
-        call_kwargs = self.mock_kube_client.read_namespaced_pod_log.call_args_list[0].kwargs
-        assert call_kwargs['since_seconds'] == 5
+        args, kwargs = self.mock_kube_client.read_namespaced_pod_log.call_args_list[0]
+        assert kwargs['since_seconds'] == 5
 
     @pytest.mark.parametrize('follow, is_running_calls, exp_running', [(True, 3, False), (False, 1, True)])
     @mock.patch('airflow.providers.cncf.kubernetes.utils.pod_manager.container_is_running')

